packages/opencode/src/cli/cmd/tui/routes/session/index.tsx

```
@@ -944,27 +947,36 @@ const PART_MAPPING = {
   reasoning: ReasoningPart,
 }
 
-function ReasoningPart(props: { part: ReasoningPart; message: AssistantMessage }) {
-  const { theme } = useTheme()
+function ReasoningPart(props: { last: boolean; part: ReasoningPart; message: AssistantMessage }) {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'ReasoningPart'. Consider to delete it or rename it.

'ReasoningPart' is defined here:


(lint/suspicious/noRedeclare)